sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -279,19 +281,87 @@ export class CallbackFunction<E, R> extends LambdaFunction {
                 assumeRolePolicy: JSON.stringify(lambdaRolePolicy),
             }, opts);
 
-            if (!args.policies) {
-                // Provides wide access to "serverless" services (Dynamo, S3, etc.)
-                args.policies = [iam.ManagedPolicy.AWSLambdaFullAccess];
-            }
-
-            for (const policy of args.policies) {
-                // RolePolicyAttachment objects don't have a physical identity, and create/deletes are processed
-                // structurally based on the `role` and `policyArn`.  So we need to make sure our Pulumi name matches the
-                // structural identity by using a name that includes the role name and policyArn.
-                const attachment = new iam.RolePolicyAttachment(`${name}-${utils.sha1hash(policy)}`, {
-                    role: role,
-                    policyArn: policy,
-                }, opts);
+            // we are going to create a copy of the "arn:aws:iam::aws:policy/AWSLambdaFullAccess" policy to ensure
+            // our mixins continue to work as expected
+            const lambdaFullAccessCopy = new iam.Policy(`${name}-LambdaFullAccess`, {
```

Comment:
This is going to create the policy no matter what. Instead, this creation should be within the conditional that previously existed `if (!args.policies)`. So this should only be created and attached if no policies are provided to the `CallbackFunction`.